README.md

```diff
@@ -67,10 +67,24 @@ $ curl -s http://localhost:9000/irods-http-api/<version>/collections \
 
 API documentation can be found in [API.md](./API.md).
 
+## iRODS Server Compatibility
+
+> [!IMPORTANT]
+> The HTTP API has not reached 1.0 status yet. It is considered unstable and can experience significant changes. While the current design has held up well, users need to be prepared to address breaking changes as needed. The **Changed** and **Removed** sections of the release notes can assist you in identifying such changes.
+
+The HTTP API is fully compatible with iRODS 4.3.2 and later.
```

Comment:
Is there partial compatibility with other versions (e.g. 4.2.12)?

Reply:
Okay. Is there an issue for that? The word "fully" is what made me think of the question about certain versions being partially compatible, if that helps.

Reply:
all releases are partially compatible with older things... the protocol has largely stayed the same.   i'm not sure it's worth trying to describe/articulate those differences... that's the constable full-matrix...

Reply:
Removed "fully".

Reply:
Opened #463 to address the 4.2 compatibility config option.